### PR TITLE
Volume browser

### PR DESCRIPTION
**Pull request by XK4MiLX**
Volume Browser Features:
- upload multiple file
- download multiple file
- rename file/folder
- craeate directory
- remove file/folder
- volume storage info

### CODE REVIEW

File: HomeUI/src/views/apps/Management.vue [hunk 1]
```diff
@@ -8081,18 +8761,6 @@ export default {
       });
       return string;
     },
-
-    showToast(variant, title, icon = 'InfoIcon') {
```
Comment (Cabecinha84) [inline HomeUI/src/views/apps/Management.vue:8085]: this function was deleted by mistake?

File: HomeUI/src/views/apps/Management.vue [hunk 2]
```diff
@@ -7400,12 +8087,6 @@ export default {
         this.showToast('danger', error.message || error);
       }
     },
-
-    cancelDownload() {
```
Comment (Cabecinha84) [inline HomeUI/src/views/apps/Management.vue:7404]: this function was deleted by mistake?

File: HomeUI/src/views/apps/Management.vue [hunk 3]
```diff
@@ -5441,7 +5767,6 @@ export default {
           ],
         },
       ],
-      timeoptions,
```
Comment (Cabecinha84) [inline HomeUI/src/views/apps/Management.vue:5444]: timeoptions is not used anywhere?

File: HomeUI/src/views/apps/Management.vue [hunk 4]
```diff
@@ -5518,10 +5843,7 @@ export default {
         totalRows: 1,
         currentPage: 1,
       },
-      total: '',
-      downloaded: '',
       downloadedSize: '',
-      abortToken: {},
```
Comment (Cabecinha84) [inline HomeUI/src/views/apps/Management.vue:5524]: this 3 deleted properties are no longer used?

**Review by Cabecinha84 — CHANGES_REQUESTED** [review]
Check my comments. Thank you

File: HomeUI/src/views/apps/Management.vue (hunk 4 above)
Comment (XK4MiLX) [reply HomeUI/src/views/apps/Management.vue:5524]: duplication

File: HomeUI/src/views/apps/Management.vue (hunk 2 above)
Comment (XK4MiLX) [reply HomeUI/src/views/apps/Management.vue:7404]: duplication

**Pull request closed without merging**